### PR TITLE
style attribute update logic optimized

### PR DESCRIPTION
**Pull request by briefy**
I changed the style attribute update logic,
it is easier to understand and less code

### CODE REVIEW

File: src/dom/index.js [hunk 1]
```diff
@@ -47,13 +47,11 @@ export function setAccessor(node, name, old, value, isSvg) {
...
-		if (!value || typeof value==='string' || typeof old==='string') {
+		if (!value || typeof value === 'string') {
 			node.style.cssText = value || '';
 		}
-		if (value && typeof value==='object') {
-			if (typeof old!=='string') {
-				for (let i in old) if (!(i in value)) node.style[i] = '';
-			}
+		if (value && typeof value === 'object') {
+			node.style.cssText = '';
```
Comment (developit) [inline src/dom/index.js]: This wipes CSS on every render, which would be very expensive.

**Comment by developit** [comment]
I pulled the performance numbers from the two Travis builds and it looks like the impact is quite severe:

| Before | After
|---|----|
| screen shot 2017-07-01 at 3 03 49 pm | screen shot 2017-07-01 at 3 03 37 pm

**Comment by briefy** [comment]
@developit 
I wonder why the empty diff drop so much, I see the performance test,
and on my computer,  I don't see such a drop.

**Comment by marvinhagemeister** [comment]
Looks like this PR is abandoned.

**Pull request closed without merging**